code/game/objects/items/devices/traitordevices.dm

```diff
@@ -286,6 +286,39 @@
 	to_chat(user, "<span class='notice'>You inject yourself with the nanites!</span>")
 	ADD_TRAIT(user, TRAIT_RESISTHEAT, "fireproof_injector")
 
+/obj/item/cryoregenerative_enhancer
+	name = "cryoregenerative enhancer"
+	desc = "Specially designed nanomachines that enhance the low-temperature regenerative capabilities of drask. Requires supercooled air in the enviroment or internals to function."
+	icon = 'icons/obj/hypo.dmi'
+	icon_state = "combat_hypo"
+	var/used = FALSE
+
+/obj/item/cryoregenerative_enhancer/examine_more(mob/user)
+	. = ..()
+	. = list()
```

Comment:
```suggestion
```
Not needed unless you have a subtype of the injector with a different description.